## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 509438`
Commit: `28db2bb9b3ded6cebb0ffc49b4a2c994419eeac1`

---
### `aarch64-darwin`
<details>
  <summary>:x: 32 packages failed to build:</summary>
  <ul>
    <li>aider-chat-full</li>
    <li>aider-chat-full.dist</li>
    <li>aider-chat-with-browser</li>
    <li>aider-chat-with-browser.dist</li>
    <li>aider-chat-with-help</li>
    <li>aider-chat-with-help.dist</li>
    <li>ha-mcp</li>
    <li>ha-mcp.dist</li>
    <li>mcp-nixos</li>
    <li>mcp-nixos.dist</li>
    <li>oterm</li>
    <li>oterm.dist</li>
    <li>python313Packages.arviz-plots</li>
    <li>python313Packages.arviz-plots.dist</li>
    <li>python313Packages.ax-platform</li>
    <li>python313Packages.ax-platform.dist</li>
    <li>python313Packages.fastmcp</li>
    <li>python313Packages.fastmcp.dist</li>
    <li>python313Packages.langchain-community</li>
    <li>python313Packages.langchain-community.dist</li>
    <li>python313Packages.langchain-experimental</li>
    <li>python313Packages.langchain-experimental.dist</li>
    <li>python313Packages.langchain-mongodb</li>
    <li>python313Packages.langchain-mongodb.dist</li>
    <li>python313Packages.langgraph-checkpoint-mongodb</li>
    <li>python313Packages.langgraph-checkpoint-mongodb.dist</li>
    <li>python313Packages.langgraph-store-mongodb</li>
    <li>python313Packages.langgraph-store-mongodb.dist</li>
    <li>python314Packages.arviz-plots</li>
    <li>python314Packages.arviz-plots.dist</li>
    <li>python314Packages.dissect-cobaltstrike</li>
    <li>python314Packages.dissect-cobaltstrike.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 513 packages built:</summary>
  <ul>
    <li>acquire (python313Packages.acquire)</li>
    <li>acquire.dist (python313Packages.acquire.dist)</li>
    <li>apkleaks</li>
    <li>apkleaks.dist</li>
    <li>azure-cli-extensions.rdbms-connect</li>
    <li>azure-cli-extensions.rdbms-connect.dist</li>
    <li>checkov</li>
    <li>checkov.dist</li>
    <li>cve-bin-tool</li>
    <li>cve-bin-tool.dist</li>
    <li>das</li>
    <li>das.dist</li>
    <li>deeptools</li>
    <li>deeptools.dist</li>
    <li>duckdb</li>
    <li>duckdb.dev</li>
    <li>duckdb.lib</li>
    <li>flatten-references-graph</li>
    <li>flatten-references-graph.dist</li>
    <li>harlequin</li>
    <li>harlequin.dist</li>
    <li>heretic (python313Packages.heretic-llm)</li>
    <li>heretic.dist (python313Packages.heretic-llm.dist)</li>
    <li>jadx</li>
    <li>libretranslate (python313Packages.libretranslate)</li>
    <li>libretranslate.dist (python313Packages.libretranslate.dist)</li>
    <li>marimo (python313Packages.marimo)</li>
    <li>marimo.dist (python313Packages.marimo.dist)</li>
    <li>multiqc</li>
    <li>multiqc.dist</li>
    <li>mycli</li>
    <li>mycli.dist</li>
    <li>prowler</li>
    <li>prowler.dist</li>
    <li>py-wacz (python313Packages.wacz)</li>
    <li>py-wacz.dist (python313Packages.wacz.dist)</li>
    <li>python313Packages.altair</li>
    <li>python313Packages.altair.dist</li>
    <li>python313Packages.amd-quark</li>
    <li>python313Packages.amd-quark.dist</li>
    <li>python313Packages.argos-translate-files</li>
    <li>python313Packages.argos-translate-files.dist</li>
    <li>python313Packages.argostranslate</li>
    <li>python313Packages.argostranslate.dist</li>
    <li>python313Packages.backtesting</li>
    <li>python313Packages.backtesting.dist</li>
    <li>python313Packages.bokeh</li>
    <li>python313Packages.bokeh.dist</li>
    <li>python313Packages.bumps</li>
    <li>python313Packages.bumps.dist</li>
    <li>python313Packages.cltk</li>
    <li>python313Packages.cltk.dist</li>
    <li>python313Packages.coiled</li>
    <li>python313Packages.coiled.dist</li>
    <li>python313Packages.compressai</li>
    <li>python313Packages.compressai.dist</li>
    <li>python313Packages.csvw</li>
    <li>python313Packages.csvw.dist</li>
    <li>python313Packages.dash</li>
    <li>python313Packages.dash-bootstrap-components</li>
    <li>python313Packages.dash-bootstrap-components.dist</li>
    <li>python313Packages.dash-bootstrap-templates</li>
    <li>python313Packages.dash-bootstrap-templates.dist</li>
    <li>python313Packages.dash.dist</li>
    <li>python313Packages.dissect</li>
    <li>python313Packages.dissect-cobaltstrike</li>
    <li>python313Packages.dissect-cobaltstrike.dist</li>
    <li>python313Packages.dissect-target</li>
    <li>python313Packages.dissect-target.dist</li>
    <li>python313Packages.dissect.dist</li>
    <li>python313Packages.duckdb</li>
    <li>python313Packages.duckdb-engine</li>
    <li>python313Packages.duckdb-engine.dist</li>
    <li>python313Packages.duckdb.dist</li>
    <li>python313Packages.energyflow</li>
    <li>python313Packages.energyflow.dist</li>
    <li>python313Packages.experiment-utilities</li>
    <li>python313Packages.experiment-utilities.dist</li>
    <li>python313Packages.explorerscript</li>
    <li>python313Packages.explorerscript.dist</li>
    <li>python313Packages.extra-streamlit-components</li>
    <li>python313Packages.extra-streamlit-components.dist</li>
    <li>python313Packages.fastai</li>
    <li>python313Packages.fastai.dist</li>
    <li>python313Packages.flow-record</li>
    <li>python313Packages.flow-record.dist</li>
    <li>python313Packages.formulaic</li>
    <li>python313Packages.formulaic.dist</li>
    <li>python313Packages.frictionless</li>
    <li>python313Packages.frictionless.dist</li>
    <li>python313Packages.gradio</li>
    <li>python313Packages.gradio.dist</li>
    <li>python313Packages.great-tables</li>
    <li>python313Packages.great-tables.dist</li>
    <li>python313Packages.hakuin</li>
    <li>python313Packages.hakuin.dist</li>
    <li>python313Packages.holoviews</li>
    <li>python313Packages.holoviews.dist</li>
    <li>python313Packages.hvplot</li>
    <li>python313Packages.hvplot.dist</li>
    <li>python313Packages.ibis-framework</li>
    <li>python313Packages.ibis-framework.dist</li>
    <li>python313Packages.igraph</li>
    <li>python313Packages.igraph.dist</li>
    <li>python313Packages.intake</li>
    <li>python313Packages.intake.dist</li>
    <li>python313Packages.iplotx</li>
    <li>python313Packages.iplotx.dist</li>
    <li>python313Packages.itables</li>
    <li>python313Packages.itables.dist</li>
    <li>python313Packages.jupysql</li>
    <li>python313Packages.jupysql.dist</li>
    <li>python313Packages.k-diffusion</li>
    <li>python313Packages.k-diffusion.dist</li>
    <li>python313Packages.kmapper</li>
    <li>python313Packages.kmapper.dist</li>
    <li>python313Packages.lancedb</li>
    <li>python313Packages.lancedb.dist</li>
    <li>python313Packages.layoutparser</li>
    <li>python313Packages.layoutparser.dist</li>
    <li>python313Packages.leidenalg</li>
    <li>python313Packages.leidenalg.dist</li>
    <li>python313Packages.lifelines</li>
    <li>python313Packages.lifelines.dist</li>
    <li>python313Packages.linearmodels</li>
    <li>python313Packages.linearmodels.dist</li>
    <li>python313Packages.livelossplot</li>
    <li>python313Packages.livelossplot.dist</li>
    <li>python313Packages.llama-cloud</li>
    <li>python313Packages.llama-cloud.dist</li>
    <li>python313Packages.llama-index</li>
    <li>python313Packages.llama-index-cli</li>
    <li>python313Packages.llama-index-cli.dist</li>
    <li>python313Packages.llama-index-core</li>
    <li>python313Packages.llama-index-core.dist</li>
    <li>python313Packages.llama-index-embeddings-google-genai</li>
    <li>python313Packages.llama-index-embeddings-google-genai.dist</li>
    <li>python313Packages.llama-index-embeddings-huggingface</li>
    <li>python313Packages.llama-index-embeddings-huggingface.dist</li>
    <li>python313Packages.llama-index-embeddings-ollama</li>
    <li>python313Packages.llama-index-embeddings-ollama.dist</li>
    <li>python313Packages.llama-index-embeddings-openai</li>
    <li>python313Packages.llama-index-embeddings-openai.dist</li>
    <li>python313Packages.llama-index-graph-stores-neo4j</li>
    <li>python313Packages.llama-index-graph-stores-neo4j.dist</li>
    <li>python313Packages.llama-index-graph-stores-neptune</li>
    <li>python313Packages.llama-index-graph-stores-neptune.dist</li>
    <li>python313Packages.llama-index-legacy</li>
    <li>python313Packages.llama-index-legacy.dist</li>
    <li>python313Packages.llama-index-llms-ollama</li>
    <li>python313Packages.llama-index-llms-ollama.dist</li>
    <li>python313Packages.llama-index-llms-openai</li>
    <li>python313Packages.llama-index-llms-openai-like</li>
    <li>python313Packages.llama-index-llms-openai-like.dist</li>
    <li>python313Packages.llama-index-llms-openai.dist</li>
    <li>python313Packages.llama-index-node-parser-docling</li>
    <li>python313Packages.llama-index-node-parser-docling.dist</li>
    <li>python313Packages.llama-index-readers-database</li>
    <li>python313Packages.llama-index-readers-database.dist</li>
    <li>python313Packages.llama-index-readers-file</li>
    <li>python313Packages.llama-index-readers-file.dist</li>
    <li>python313Packages.llama-index-readers-json</li>
    <li>python313Packages.llama-index-readers-json.dist</li>
    <li>python313Packages.llama-index-readers-s3</li>
    <li>python313Packages.llama-index-readers-s3.dist</li>
    <li>python313Packages.llama-index-readers-twitter</li>
    <li>python313Packages.llama-index-readers-twitter.dist</li>
    <li>python313Packages.llama-index-readers-txtai</li>
    <li>python313Packages.llama-index-readers-txtai.dist</li>
    <li>python313Packages.llama-index-readers-weather</li>
    <li>python313Packages.llama-index-readers-weather.dist</li>
    <li>python313Packages.llama-index-vector-stores-chroma</li>
    <li>python313Packages.llama-index-vector-stores-chroma.dist</li>
    <li>python313Packages.llama-index-vector-stores-google</li>
    <li>python313Packages.llama-index-vector-stores-google.dist</li>
    <li>python313Packages.llama-index-vector-stores-milvus</li>
    <li>python313Packages.llama-index-vector-stores-milvus.dist</li>
    <li>python313Packages.llama-index-vector-stores-qdrant</li>
    <li>python313Packages.llama-index-vector-stores-qdrant.dist</li>
    <li>python313Packages.llama-index.dist</li>
    <li>python313Packages.mlcroissant</li>
    <li>python313Packages.mlcroissant.dist</li>
    <li>python313Packages.msticpy</li>
    <li>python313Packages.msticpy.dist</li>
    <li>python313Packages.narwhals</li>
    <li>python313Packages.narwhals.dist</li>
    <li>python313Packages.nbsmoke</li>
    <li>python313Packages.nbsmoke.dist</li>
    <li>python313Packages.neurokit2</li>
    <li>python313Packages.neurokit2.dist</li>
    <li>python313Packages.niaarm</li>
    <li>python313Packages.niaarm.dist</li>
    <li>python313Packages.niworkflows</li>
    <li>python313Packages.niworkflows.dist</li>
    <li>python313Packages.optuna</li>
    <li>python313Packages.optuna-dashboard</li>
    <li>python313Packages.optuna-dashboard.dist</li>
    <li>python313Packages.optuna.dist</li>
    <li>python313Packages.pandantic</li>
    <li>python313Packages.pandantic.dist</li>
    <li>python313Packages.pandas-stubs</li>
    <li>python313Packages.pandas-stubs.dist</li>
    <li>python313Packages.pandera</li>
    <li>python313Packages.pandera.dist</li>
    <li>python313Packages.panel</li>
    <li>python313Packages.panel.dist</li>
    <li>python313Packages.pdfplumber</li>
    <li>python313Packages.pdfplumber.dist</li>
    <li>python313Packages.plopp</li>
    <li>python313Packages.plopp.dist</li>
    <li>python313Packages.plotly</li>
    <li>python313Packages.plotly.dist</li>
    <li>python313Packages.py-key-value-aio</li>
    <li>python313Packages.py-key-value-aio.dist</li>
    <li>python313Packages.pyannote-audio</li>
    <li>python313Packages.pyannote-audio.dist</li>
    <li>python313Packages.pyannote-pipeline</li>
    <li>python313Packages.pyannote-pipeline.dist</li>
    <li>python313Packages.pybids</li>
    <li>python313Packages.pybids.dist</li>
    <li>python313Packages.pydocket</li>
    <li>python313Packages.pydocket.dist</li>
    <li>python313Packages.pylance</li>
    <li>python313Packages.pylance.dist</li>
    <li>python313Packages.pymoo</li>
    <li>python313Packages.pymoo.dist</li>
    <li>python313Packages.pyreadstat</li>
    <li>python313Packages.pyreadstat.dist</li>
    <li>python313Packages.reflex</li>
    <li>python313Packages.reflex-chakra</li>
    <li>python313Packages.reflex-chakra.dist</li>
    <li>python313Packages.reflex.dist</li>
    <li>python313Packages.sasmodels</li>
    <li>python313Packages.sasmodels.dist</li>
    <li>python313Packages.scikit-tda</li>
    <li>python313Packages.scikit-tda.dist</li>
    <li>python313Packages.segments</li>
    <li>python313Packages.segments.dist</li>
    <li>python313Packages.shiny</li>
    <li>python313Packages.shiny.dist</li>
    <li>python313Packages.skops</li>
    <li>python313Packages.skops.dist</li>
    <li>python313Packages.smolagents</li>
    <li>python313Packages.smolagents.dist</li>
    <li>python313Packages.spacy</li>
    <li>python313Packages.spacy-curated-transformers</li>
    <li>python313Packages.spacy-curated-transformers.dist</li>
    <li>python313Packages.spacy-loggers</li>
    <li>python313Packages.spacy-loggers.dist</li>
    <li>python313Packages.spacy-lookups-data</li>
    <li>python313Packages.spacy-lookups-data.dist</li>
    <li>python313Packages.spacy-transformers</li>
    <li>python313Packages.spacy-transformers.dist</li>
    <li>python313Packages.spacy.dist</li>
    <li>python313Packages.sqlframe</li>
    <li>python313Packages.sqlframe.dist</li>
    <li>python313Packages.sqlglot</li>
    <li>python313Packages.sqlglot.dist</li>
    <li>python313Packages.st-pages</li>
    <li>python313Packages.st-pages.dist</li>
    <li>streamlit (python313Packages.streamlit)</li>
    <li>python313Packages.streamlit-avatar</li>
    <li>python313Packages.streamlit-avatar.dist</li>
    <li>python313Packages.streamlit-card</li>
    <li>python313Packages.streamlit-card.dist</li>
    <li>python313Packages.streamlit-echarts</li>
    <li>python313Packages.streamlit-echarts.dist</li>
    <li>python313Packages.streamlit-folium</li>
    <li>python313Packages.streamlit-folium.dist</li>
    <li>python313Packages.streamlit-kpi-card</li>
    <li>python313Packages.streamlit-kpi-card.dist</li>
    <li>python313Packages.streamlit-notify</li>
    <li>python313Packages.streamlit-notify.dist</li>
    <li>streamlit.dist (python313Packages.streamlit.dist)</li>
    <li>python313Packages.svgdigitizer</li>
    <li>python313Packages.svgdigitizer.dist</li>
    <li>python313Packages.synergy</li>
    <li>python313Packages.synergy.dist</li>
    <li>python313Packages.tcia-utils</li>
    <li>python313Packages.tcia-utils.dist</li>
    <li>python313Packages.templateflow</li>
    <li>python313Packages.templateflow.dist</li>
    <li>python313Packages.textacy</li>
    <li>python313Packages.textacy.dist</li>
    <li>python313Packages.textnets</li>
    <li>python313Packages.textnets.dist</li>
    <li>python313Packages.translatehtml</li>
    <li>python313Packages.translatehtml.dist</li>
    <li>python313Packages.unstructured-inference</li>
    <li>python313Packages.unstructured-inference.dist</li>
    <li>python313Packages.vega</li>
    <li>python313Packages.vega.dist</li>
    <li>python313Packages.vegafusion</li>
    <li>python313Packages.vegafusion.dist</li>
    <li>python313Packages.wandb</li>
    <li>python313Packages.wandb.dist</li>
    <li>whisperx (python313Packages.whisperx)</li>
    <li>whisperx.dist (python313Packages.whisperx.dist)</li>
    <li>python314Packages.altair</li>
    <li>python314Packages.altair.dist</li>
    <li>python314Packages.amd-quark</li>
    <li>python314Packages.amd-quark.dist</li>
    <li>python314Packages.ax-platform</li>
    <li>python314Packages.ax-platform.dist</li>
    <li>python314Packages.backtesting</li>
    <li>python314Packages.backtesting.dist</li>
    <li>python314Packages.bokeh</li>
    <li>python314Packages.bokeh.dist</li>
    <li>python314Packages.bumps</li>
    <li>python314Packages.bumps.dist</li>
    <li>python314Packages.coiled</li>
    <li>python314Packages.coiled.dist</li>
    <li>python314Packages.compressai</li>
    <li>python314Packages.compressai.dist</li>
    <li>python314Packages.csvw</li>
    <li>python314Packages.csvw.dist</li>
    <li>python314Packages.dash</li>
    <li>python314Packages.dash-bootstrap-components</li>
    <li>python314Packages.dash-bootstrap-components.dist</li>
    <li>python314Packages.dash-bootstrap-templates</li>
    <li>python314Packages.dash-bootstrap-templates.dist</li>
    <li>python314Packages.dash.dist</li>
    <li>python314Packages.duckdb</li>
    <li>python314Packages.duckdb-engine</li>
    <li>python314Packages.duckdb-engine.dist</li>
    <li>python314Packages.duckdb.dist</li>
    <li>python314Packages.experiment-utilities</li>
    <li>python314Packages.experiment-utilities.dist</li>
    <li>python314Packages.explorerscript</li>
    <li>python314Packages.explorerscript.dist</li>
    <li>python314Packages.extra-streamlit-components</li>
    <li>python314Packages.extra-streamlit-components.dist</li>
    <li>python314Packages.flow-record</li>
    <li>python314Packages.flow-record.dist</li>
    <li>python314Packages.formulaic</li>
    <li>python314Packages.formulaic.dist</li>
    <li>python314Packages.frictionless</li>
    <li>python314Packages.frictionless.dist</li>
    <li>python314Packages.gradio</li>
    <li>python314Packages.gradio.dist</li>
    <li>python314Packages.great-tables</li>
    <li>python314Packages.great-tables.dist</li>
    <li>python314Packages.hakuin</li>
    <li>python314Packages.hakuin.dist</li>
    <li>python314Packages.harlequin-postgres</li>
    <li>python314Packages.harlequin-postgres.dist</li>
    <li>python314Packages.heretic-llm</li>
    <li>python314Packages.heretic-llm.dist</li>
    <li>python314Packages.holoviews</li>
    <li>python314Packages.holoviews.dist</li>
    <li>python314Packages.hvplot</li>
    <li>python314Packages.hvplot.dist</li>
    <li>python314Packages.ibis-framework</li>
    <li>python314Packages.ibis-framework.dist</li>
    <li>python314Packages.igraph</li>
    <li>python314Packages.igraph.dist</li>
    <li>python314Packages.intake</li>
    <li>python314Packages.intake.dist</li>
    <li>python314Packages.iplotx</li>
    <li>python314Packages.iplotx.dist</li>
    <li>python314Packages.itables</li>
    <li>python314Packages.itables.dist</li>
    <li>python314Packages.jupysql</li>
    <li>python314Packages.jupysql.dist</li>
    <li>python314Packages.k-diffusion</li>
    <li>python314Packages.k-diffusion.dist</li>
    <li>python314Packages.kmapper</li>
    <li>python314Packages.kmapper.dist</li>
    <li>python314Packages.lancedb</li>
    <li>python314Packages.lancedb.dist</li>
    <li>python314Packages.langchain-community</li>
    <li>python314Packages.langchain-community.dist</li>
    <li>python314Packages.langchain-experimental</li>
    <li>python314Packages.langchain-experimental.dist</li>
    <li>python314Packages.langchain-mongodb</li>
    <li>python314Packages.langchain-mongodb.dist</li>
    <li>python314Packages.langgraph-checkpoint-mongodb</li>
    <li>python314Packages.langgraph-checkpoint-mongodb.dist</li>
    <li>python314Packages.langgraph-store-mongodb</li>
    <li>python314Packages.langgraph-store-mongodb.dist</li>
    <li>python314Packages.layoutparser</li>
    <li>python314Packages.layoutparser.dist</li>
    <li>python314Packages.leidenalg</li>
    <li>python314Packages.leidenalg.dist</li>
    <li>python314Packages.lifelines</li>
    <li>python314Packages.lifelines.dist</li>
    <li>python314Packages.linearmodels</li>
    <li>python314Packages.linearmodels.dist</li>
    <li>python314Packages.livelossplot</li>
    <li>python314Packages.livelossplot.dist</li>
    <li>python314Packages.marimo</li>
    <li>python314Packages.marimo.dist</li>
    <li>python314Packages.mlcroissant</li>
    <li>python314Packages.mlcroissant.dist</li>
    <li>python314Packages.msticpy</li>
    <li>python314Packages.msticpy.dist</li>
    <li>python314Packages.narwhals</li>
    <li>python314Packages.narwhals.dist</li>
    <li>python314Packages.nbsmoke</li>
    <li>python314Packages.nbsmoke.dist</li>
    <li>python314Packages.neurokit2</li>
    <li>python314Packages.neurokit2.dist</li>
    <li>python314Packages.niaarm</li>
    <li>python314Packages.niaarm.dist</li>
    <li>python314Packages.niworkflows</li>
    <li>python314Packages.niworkflows.dist</li>
    <li>python314Packages.optuna</li>
    <li>python314Packages.optuna-dashboard</li>
    <li>python314Packages.optuna-dashboard.dist</li>
    <li>python314Packages.optuna.dist</li>
    <li>python314Packages.pandantic</li>
    <li>python314Packages.pandantic.dist</li>
    <li>python314Packages.pandas-stubs</li>
    <li>python314Packages.pandas-stubs.dist</li>
    <li>python314Packages.pandera</li>
    <li>python314Packages.pandera.dist</li>
    <li>python314Packages.panel</li>
    <li>python314Packages.panel.dist</li>
    <li>python314Packages.pdfplumber</li>
    <li>python314Packages.pdfplumber.dist</li>
    <li>python314Packages.plopp</li>
    <li>python314Packages.plopp.dist</li>
    <li>python314Packages.plotly</li>
    <li>python314Packages.plotly.dist</li>
    <li>python314Packages.pyannote-audio</li>
    <li>python314Packages.pyannote-audio.dist</li>
    <li>python314Packages.pyannote-pipeline</li>
    <li>python314Packages.pyannote-pipeline.dist</li>
    <li>python314Packages.pybids</li>
    <li>python314Packages.pybids.dist</li>
    <li>python314Packages.pylance</li>
    <li>python314Packages.pylance.dist</li>
    <li>python314Packages.pymoo</li>
    <li>python314Packages.pymoo.dist</li>
    <li>python314Packages.pyreadstat</li>
    <li>python314Packages.pyreadstat.dist</li>
    <li>python314Packages.reflex</li>
    <li>python314Packages.reflex-chakra</li>
    <li>python314Packages.reflex-chakra.dist</li>
    <li>python314Packages.reflex.dist</li>
    <li>python314Packages.sasmodels</li>
    <li>python314Packages.sasmodels.dist</li>
    <li>python314Packages.scikit-tda</li>
    <li>python314Packages.scikit-tda.dist</li>
    <li>python314Packages.segments</li>
    <li>python314Packages.segments.dist</li>
    <li>python314Packages.shiny</li>
    <li>python314Packages.shiny.dist</li>
    <li>python314Packages.skops</li>
    <li>python314Packages.skops.dist</li>
    <li>python314Packages.smolagents</li>
    <li>python314Packages.smolagents.dist</li>
    <li>python314Packages.spacy-loggers</li>
    <li>python314Packages.spacy-loggers.dist</li>
    <li>python314Packages.sqlframe</li>
    <li>python314Packages.sqlframe.dist</li>
    <li>python314Packages.sqlglot</li>
    <li>python314Packages.sqlglot.dist</li>
    <li>python314Packages.st-pages</li>
    <li>python314Packages.st-pages.dist</li>
    <li>python314Packages.streamlit</li>
    <li>python314Packages.streamlit-avatar</li>
    <li>python314Packages.streamlit-avatar.dist</li>
    <li>python314Packages.streamlit-card</li>
    <li>python314Packages.streamlit-card.dist</li>
    <li>python314Packages.streamlit-echarts</li>
    <li>python314Packages.streamlit-echarts.dist</li>
    <li>python314Packages.streamlit-folium</li>
    <li>python314Packages.streamlit-folium.dist</li>
    <li>python314Packages.streamlit-kpi-card</li>
    <li>python314Packages.streamlit-kpi-card.dist</li>
    <li>python314Packages.streamlit-notify</li>
    <li>python314Packages.streamlit-notify.dist</li>
    <li>python314Packages.streamlit.dist</li>
    <li>python314Packages.svgdigitizer</li>
    <li>python314Packages.svgdigitizer.dist</li>
    <li>python314Packages.synergy</li>
    <li>python314Packages.synergy.dist</li>
    <li>python314Packages.tcia-utils</li>
    <li>python314Packages.tcia-utils.dist</li>
    <li>python314Packages.templateflow</li>
    <li>python314Packages.templateflow.dist</li>
    <li>python314Packages.unstructured-inference</li>
    <li>python314Packages.unstructured-inference.dist</li>
    <li>python314Packages.vega</li>
    <li>python314Packages.vega.dist</li>
    <li>python314Packages.vegafusion</li>
    <li>python314Packages.vegafusion.dist</li>
    <li>python314Packages.wacz</li>
    <li>python314Packages.wacz.dist</li>
    <li>python314Packages.wandb</li>
    <li>python314Packages.wandb.dist</li>
    <li>python314Packages.whisperx</li>
    <li>python314Packages.whisperx.dist</li>
    <li>quark-engine</li>
    <li>quark-engine.dist</li>
    <li>rembg</li>
    <li>rembg.dist</li>
    <li>sptk</li>
    <li>sqlit-tui</li>
    <li>sqlit-tui.dist</li>
    <li>sqlite3-to-mysql</li>
    <li>sqlite3-to-mysql.dist</li>
    <li>theharvester</li>
    <li>theharvester.dist</li>
    <li>tocpdf</li>
    <li>tocpdf.dist</li>
    <li>tts</li>
    <li>tts.dist</li>
    <li>unstructured-api</li>
    <li>vimPlugins.nvim-dbee</li>
    <li>whisper-ctranslate2</li>
    <li>whisper-ctranslate2.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>python313Packages.arviz-plots</summary>
<pre>    │       Data variables:
    │           mu         (ci_bound) float64 16B 0.4389 1.671
    │           theta      (hierarchy, ci_bound) float64 112B -0.07926 1.204 ... 0.7839
    │           tau        (ci_bound) float64 16B 0.4849 2.251
    └── Group: /twig
            Dimensions:    (ci_bound: 2, hierarchy: 7)
            Coordinates:
              * ci_bound   (ci_bound) &lt;U5 40B &#x27;lower&#x27; &#x27;upper&#x27;
              * hierarchy  (hierarchy) int64 56B 0 1 2 3 4 5 6
            Data variables:
                mu         (ci_bound) float64 16B -0.5305 2.531
                theta      (hierarchy, ci_bound) float64 112B -0.9648 2.121 ... -1.505 1.793
                tau        (ci_bound) float64 16B 0.1426 5.474,
    visuals={},
    compact=False,
    combined=False,
)

You can reproduce this example by temporarily adding @reproduce_failure(&#x27;6.150.2&#x27;, b&#x27;AAAAAA==&#x27;) as a decorator on your test case
============ 6 failed, 606 passed, 4 skipped in 1682.03s (0:28:02) =============</pre>
</details>
<details>
<summary>python313Packages.ax-platform</summary>
<pre>  /nix/var/nix/builds/nix-13151-18906683/source/ax/storage/sqa_store/load.py:406: SAWarning: TypeDecorator JSONEncodedText() will not produce a cache key because the ``cache_ok`` attribute is not set to True.  This can have significant performance implications including some performance degradations in comparison to prior SQLAlchemy versions.  Set this attribute to True if this type object&#x27;s state is safe to use in a cache key, or False to disable this warning. (Background on this warning at: https://sqlalche.me/e/20/cprf)
    .one_or_none()

ax/utils/stats/tests/test_math_utils.py::RelativizeTest::test_relativize_with_covariance
  /nix/var/nix/builds/nix-13151-18906683/source/ax/utils/stats/math_utils.py:115: RuntimeWarning: invalid value encountered in sqrt
    return (r_hat, np.sqrt(var))

ax/utils/testing/tests/test_mock.py::TestMock::test_optimize_with_nsgaii_mocks
  /nix/store/5yfk7rlv8ndwcchrz0wb0ziczymn203y-python3.13-botorch-0.17.0/lib/python3.13/site-packages/botorch/acquisition/analytic.py:343: NumericsWarning: ExpectedImprovement has known numerical issues that lead to suboptimal optimization performance. It is strongly recommended to simply replace
  
  	 ExpectedImprovement 	 --&gt; 	 LogExpectedImprovement 
  
  instead, which fixes the issues and has the same API. See https://arxiv.org/abs/2310.20708 for details.
    legacy_ei_numerics_warning(legacy_name=type(self).__name__)

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED ax/adapter/tests/test_cross_validation.py::CrossValidationTest::test_efficient_loo_cv_with_fully_bayesian_model - TimeoutError: Test took longer than 120 seconds. To see a profiler output, set `TestCase.PROFILE_TESTS` to `True`. To specify a reason for a long running test, utilize the @ax_long_test decorator. If your test is long because it&#x27;s doing modeling, please use the @mock_botorch_optimize decorator and see if that helps.
FAILED ax/adapter/tests/test_random_adapter.py::RandomAdapterTest::test_deduplicate - TimeoutError: Test took longer than 120 seconds. To see a profiler output, set `TestCase.PROFILE_TESTS` to `True`. To specify a reason for a long running test, utilize the @ax_long_test decorator. If your test is long because it&#x27;s doing modeling, please use the @mock_botorch_optimize decorator and see if that helps.
== 2 failed, 1889 passed, 147 deselected, 3017 warnings in 2655.18s (0:44:15) ==</pre>
</details>
<details>
<summary>python313Packages.fastmcp</summary>
<pre>tests/test_mcp_config.py:639
  /nix/var/nix/builds/nix-13151-18906583/source/tests/test_mcp_config.py:639: PytestUnknownMarkWarning: Unknown pytest.mark.flaky - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/how-to/mark.html
    @pytest.mark.flaky(retries=3)

tests/deprecated/test_exclude_args.py::test_tool_exclude_args_in_tool_manager
tests/deprecated/test_exclude_args.py::test_tool_exclude_args_without_default_value_raises_error
tests/deprecated/test_exclude_args.py::test_add_tool_method_exclude_args
tests/deprecated/test_exclude_args.py::test_tool_functionality_with_exclude_args
tests/deprecated/test_exclude_args.py::test_exclude_args_with_non_serializable_type
  /nix/store/rg1xmlnwgajggsmri5zljq960mdjf2b3-python3.13-fastmcp-2.14.5/lib/python3.13/site-packages/fastmcp/tools/tool.py:212: DeprecationWarning: The `exclude_args` parameter is deprecated as of FastMCP 2.14. Use dependency injection with `Depends()` instead for better lifecycle management. See https://gofastmcp.com/servers/dependencies for examples.
    return FunctionTool.from_function(

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/server/middleware/test_rate_limiting.py::TestRateLimitingMiddlewareIntegration::test_rate_limiting_blocks_rapid_requests - Failed: DID NOT RAISE &lt;class &#x27;fastmcp.exceptions.ToolError&#x27;&gt;
FAILED tests/server/middleware/test_rate_limiting.py::TestRateLimitingMiddlewareIntegration::test_rate_limiting_with_different_operations - Failed: DID NOT RAISE &lt;class &#x27;fastmcp.exceptions.ToolError&#x27;&gt;
FAILED tests/server/middleware/test_rate_limiting.py::TestRateLimitingMiddlewareIntegration::test_custom_client_identification - Failed: DID NOT RAISE &lt;class &#x27;fastmcp.exceptions.ToolError&#x27;&gt;
FAILED tests/server/middleware/test_rate_limiting.py::TestRateLimitingMiddlewareIntegration::test_rate_limiting_recovery_over_time - Failed: DID NOT RAISE &lt;class &#x27;fastmcp.exceptions.ToolError&#x27;&gt;
FAILED tests/server/openapi/test_comprehensive.py::TestOpenAPIComprehensive::test_server_performance_no_latency - assert 0.18927788734436035 &lt; 0.1
= 5 failed, 3009 passed, 104 deselected, 12 xfailed, 1 xpassed, 14 warnings in 1569.59s (0:26:09) =</pre>
</details>
<details>
<summary>python314Packages.arviz-plots</summary>
<pre>    |     raise DeadlineExceeded(
    |         datetime.timedelta(seconds=runtime), self.settings.deadline
    |     )
    | hypothesis.errors.DeadlineExceeded: Test took 4392.88ms, which exceeds the deadline of 3000.00ms. If you expect test cases to take this long, you can use @settings(deadline=...) to either set a higher deadline, or to disable it with deadline=None.
    +------------------------------------
============================= slowest 10 durations =============================
38.82s call     tests/test_hypothesis_plots.py::test_plot_trace_dist
32.74s call     tests/test_hypothesis_plots.py::test_plot_pair
27.68s call     tests/test_hypothesis_plots.py::test_plot_rank
26.40s call     tests/test_hypothesis_plots.py::test_plot_rank_dist
25.88s call     tests/test_hypothesis_plots.py::test_plot_dgof_dist
24.91s call     tests/test_hypothesis_plots.py::test_plot_dgof
18.84s call     tests/test_hypothesis_plots.py::test_plot_ecdf_pit
15.98s call     tests/test_hypothesis_plots.py::test_plot_psense
13.39s call     tests/test_hypothesis_plots.py::test_plot_ridge
10.60s call     tests/test_hypothesis_plots.py::test_plot_psense_quantities
=========================== short test summary info ============================
FAILED tests/test_hypothesis_plots.py::test_plot_dgof_dist - DeadlineExceeded(&#x27;Test took 3923.14ms, which exceeds the deadline of 3000.00ms. If you expect test cases to take this long, you can use @settings(deadline=...) to either set a higher deadline, or to disable it with deadline=None.&#x27;) [single exception in FlakyFailure]
FAILED tests/test_hypothesis_plots.py::test_plot_trace_dist - DeadlineExceeded(&#x27;Test took 4392.88ms, which exceeds the deadline of 3000.00ms. If you expect test cases to take this long, you can use @settings(deadline=...) to either set a higher deadline, or to disable it with deadline=None.&#x27;) [single exception in FlakyFailure]
============= 2 failed, 610 passed, 4 skipped in 743.82s (0:12:23) =============</pre>
</details>
</details>

--- a/pkgs/by-name/du/duckdb/package.nix
+++ b/pkgs/by-name/du/duckdb/package.nix
@@ -123,8 +123,11 @@ stdenv.mkDerivation (finalAttrs: {
           "test/sql/aggregate/aggregates/histogram_table_function.test"
         ]
         ++ lib.optionals stdenv.hostPlatform.isDarwin [
-          # SIGTRAP during iejoin tests on aarch64-darwin (with and without sandbox)
-          # iejoin implementation rewritten in 1.5.x with new parallel task scheduling
+          # UB in PhysicalRangeJoin (shared by IEJoin and PiecewiseMergeJoin) causes
+          # Apple Clang at -O3 to emit brk trap instructions on aarch64-darwin.
+          # Affects any test routing through PhysicalIEJoin (2+ inequality conditions,
+          # cardinality >= merge_join_threshold) or forcing IEJoin via debug_asof_iejoin.
+          "test/sql/join/iejoin/iejoin_issue_6314.test_slow"
           "test/sql/join/iejoin/iejoin_issue_6861.test"
           "test/sql/join/iejoin/iejoin_issue_7278.test"
           "test/sql/join/iejoin/iejoin_projection_maps.test"
@@ -138,7 +141,13 @@ stdenv.mkDerivation (finalAttrs: {
           "test/sql/join/iejoin/test_iejoin_null_keys.test"
           "test/sql/join/iejoin/test_iejoin_overlaps.test"
           "test/sql/join/iejoin/test_iejoin_predicate.test"
+          "test/sql/join/iejoin/test_iejoin_sort_tasks.test_slow"
           "test/sql/join/iejoin/test_iesemijoin.test"
+          # asof tests that loop debug_asof_iejoin=True, forcing the IEJoin path
+          "test/sql/join/asof/test_asof_join_inequalities.test"
+          "test/sql/join/asof/test_asof_join_missing.test_slow"
+          # 10240-row inequality join routing to IEJoin via plan_comparison_join.cpp
+          "test/sql/join/test_complex_range_join.test"
         ]
       );
       LD_LIBRARY_PATH = lib.optionalString stdenv.hostPlatform.isDarwin "DY" + "LD_LIBRARY_PATH";

